scripts/mixins/job_special.lua

```diff
@@ -139,6 +148,19 @@ local familyEES =
     [373] = dsp.jsa.EES_GOBLIN,  -- Goblin_Armored
 }
 
+local effectByAbility =
+{
+    [dsp.jsa.MIGHTY_STRIKES] = dsp.effect.MIGHTY_STRIKES,
+    [dsp.jsa.HUNDRED_FISTS]  = dsp.effect.HUNDRED_FISTS,
+    [dsp.jsa.MANAFONT]       = dsp.effect.MANAFONT,
+    [dsp.jsa.CHAINSPELL]     = dsp.effect.CHAINSPELL,
+    [dsp.jsa.PERFECT_DODGE]  = dsp.effect.PERFECT_DODGE,
+    [dsp.jsa.INVINCIBLE]     = dsp.effect.INVINCIBLE,
+    [dsp.jsa.BLOOD_WEAPON]   = dsp.effect.BLOOD_WEAPON,
+    [dsp.jsa.SOUL_VOICE]     = dsp.effect.SOUL_VOICE,
+    [dsp.jsa.AZURE_LORE]     = dsp.effect.AZURE_LORE,
```

Comment:
Also other like Elemental Sforzo etc just in case we wind up with non default durations (eh, not the best example since its not used by any mob yet, but it exists).